README.md

```diff
@@ -7,12 +7,42 @@
 This repository provides **[ariadne](https://inception.fandom.com/wiki/Ariadne)**, a library 
 to run and implement external recommenders for INCEpTION using Python.
 
+## Install required packages
+For installing the required packages we provide a `setup.py` to simplify the process. To install
+the basic requirements just run 
+    
+    pip install .
```

Comment:
pip install -e . is better, as then changes you make are automatically visible as it symlinks and not copies the files to the install dir, I think if you only pip install, then the package is copied and if you want to see your changes you need to run that command again.

https://stackoverflow.com/questions/42609943/what-is-the-use-case-for-pip-install-e

Reply:
Thanks for the feedback. I added the -e.